test/membrane/ownKeys.spec.js

```diff
@@ -0,0 +1,30 @@
+import createSecureEnvironment from '../../lib/browser-realm.js'
+
+let exported
+function exportData(arg) {
+  exported = exportData
+}
+
+const oneScript = `
+    class Foo {}
+    Object.defineProperty(Foo, Symbol.for('@@lockerLiveValue'), {enumerable: true});
+    exportData(Foo);
+`
+
+const twoScript = `
+    const symbols = Object.getOwnPropertySymbols(Foo);
+    exportData(symbols);
+`
+
+it('returns symbols from other namespace', () => {
+  const secureEvalOne = createSecureEnvironment(undefined, { exportData })
+  secureEvalOne(oneScript)
+  const Foo = exported
+  const secureEvalTwo = createSecureEnvironment(
+    undefined,
+    { exportData, Foo }
+  );
+  secureEvalTwo(twoScript);
+  const symbols = exported;
+  expect(symbols.length).toBe(1);
```

Comment:
Since this just errors on `ownKeys` do you even need to add a symbol and call `getOwnPropertySymbols` wouldn't it error with just a plain class like:
```js
class Foo {}
Reflect.ownKeys(Foo) // should error?
```

Reply:
Simplified it could look like:
```js
let exported
function exportData(arg) {
  exported = arg
}

const oneScript = `
    exportData(class Foo {});
`

const twoScript = `
    exportData(Reflect.ownKeys(Foo));
`

it('does not throw ownKeys trap invariant for classes or strict mode functions', () => {
  const secureEvalOne = createSecureEnvironment(undefined, { exportData })
  secureEvalOne(oneScript)
  const Foo = exported
  const secureEvalTwo = createSecureEnvironment(
    undefined,
    { exportData, Foo }
  );
  secureEvalTwo(twoScript);
  const ownKeys = exported;
  const matcher = jasmine.arrayWithExactContents(['length', 'name', 'prototype']);
  expect(ownKeys).toEqual(matcher);
});
```